src/main/java/com/spotify/confidence/FlagResolverImpl.java

```diff
@@ -0,0 +1,64 @@
+package com.spotify.confidence;
+
+import com.google.common.base.Strings;
+import com.google.common.util.concurrent.ListenableFuture;
+import com.google.protobuf.Struct;
+import com.spotify.confidence.shaded.flags.resolver.v1.*;
+import io.grpc.ManagedChannel;
+import java.io.Closeable;
+import java.io.IOException;
+import java.util.List;
+import java.util.Properties;
+import java.util.concurrent.TimeUnit;
+
+class FlagResolverImpl implements FlagResolver, Closeable {
+
+  // Deadline in seconds
+  public static final int DEADLINE_AFTER_SECONDS = 10;
+  private final ManagedChannel managedChannel;
+
+  private final String clientSecret;
+
+  private final String SDK_VERSION;
+  private static final SdkId SDK_ID = SdkId.SDK_ID_JAVA_PROVIDER;
```

Comment:
Maybe we should already now get a new SDK_ID for this so we can differentiate what's actually used in prod.